crates/benches/browser/wasm/.cargo/config.toml

```diff
@@ -1,11 +1,16 @@
 [build]
 target = "wasm32-unknown-unknown"
 
+[unstable]
+build-std = ["panic_abort", "std"]
+
 [target.wasm32-unknown-unknown]
 rustflags = [
     "-C",
     "target-feature=+atomics,+bulk-memory,+mutable-globals",
+    "-C",
+    # 4GB
+    "link-arg=--max-memory=4294967296",
```

Comment:
Did you need to add this because of running in debug mode? I think I ran into this same problem once before

Reply:
i simply copy pasted from wasm/.cargo/config.toml
i assumed it was needed